src/gui/static/src/app/components/layout/hardware-wallet/hw-dialog-base.component.ts

```diff
@@ -140,6 +150,15 @@ export class HwDialogBaseComponent<T> implements OnDestroy {
         result.link = AppConfig.hwWalletDaemonDownloadUrl;
       }
 
+      // If the operation was cancelled for inactivity, a link to open the page for wiping the
+      // Skywallet is added.
+      if (result.text === 'hardware-wallet.errors.timeout') {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/skycoin/skycoin/pullRequest?prid=7163601)
```suggestion
      if (result.text === "hardware-wallet.errors.timeout") {
```